### PR TITLE
obfs4 bridge support

### PR DESCRIPTION
**Pull request by AMBULATUR**
#48 
#54 

- [x] Torrc can handle more than one bridge, please fix LineByLineConfigurer & TorConfigurationDictionary, dictionary can handle only uniq keys.

First of all you need obfs4proxy.exe for windows 
Download tor browser - https://www.torproject.org/download/ .
or "apt-get install obfs4proxy" for *nix.

-Then copy `TorBrowser\Browser\TorBrowser\Tor\PluggableTransports\obfs4proxy.exe` path into ClientTransportPlugin string.
-Take obfs4 bridge from official site - https://bridges.torproject.org/bridges?transport=obfs4 then copy into Bridge string.
-Turn on UseBridges flag.
>"Make internet work again in your country"

ex.
 ```
TorSettings = new TorSharpTorSettings
                {
                    SocksPort = 1338,
                    ControlPort = 1339,
                    ControlPassword = "foobar",
                    UseBridges = true,
                    Bridge = "obfs4 92.75.116.165:42893 3A76222696BA38823C43521FD604189A285D9859 cert=FR1z74IEj2chaLm4ztMtjnU1X4VfAAC6m6fPVSN37IggVKmhJqH0w81b1eEyYoBYDDAyZQ iat-mode=0",
                    ClientTransportPlugin = "meek_lite,obfs2,obfs3,obfs4,scramblesuit exec D:\\obfs4proxy.exe",
                },
```
Fixes #

### CODE REVIEW

File: src/TorSharp/TorSharpTorSettings.cs [hunk 1]
```diff
@@ -85,5 +85,25 @@ public TorSharpTorSettings()
...
+        /// ex. [transport] exec [Path]
+        /// </summary>
+        public string ClientTransportPlugin { get; set; }
+
+        /// <summary>
+        /// See https://2019.www.torproject.org/docs/tor-manual-dev.html.en#Bridge for more details.
+        /// Transport plugin bridge.
+        /// ex. bridge [transport] ip:port [fingerprint] key1=val2 key2=val2 keyN=valN...
+        /// </summary>
+        public string Bridge { get; set; }
```
Comment (AMBULATUR) [inline src/TorSharp/TorSharpTorSettings.cs:106]: public string[] Bridge {get;set;} after update dictionary.

Comment (joelverhagen) [reply src/TorSharp/TorSharpTorSettings.cs:106]: String is fine. It's simpler and better matches the reality of the torrc file.

**Comment by joelverhagen** [comment]
Looks great, thanks!

**Comment by joelverhagen** [comment]
This has been release in Knapcode.TorSharp 2.3.0. 

Thanks again @AMBULATUR!

**Pull request merged**